tools/src/main/scala/scala/scalanative/build/Discover.scala

```diff
@@ -43,21 +43,33 @@ object Discover {
   /** Find default clang compilation options. */
   def compileOptions(): Seq[String] = {
     val includes = {
-      val includedir =
-        Try(Process("llvm-config --includedir").lineStream_!.toSeq)
+      val llvmIncludeDir =
+        Try(Process("llvm-config --includedir").lineStream_!)
           .getOrElse(Seq.empty)
-      ("/usr/local/include" +: includedir).map(s => s"-I$s")
+
+      val includeDirs =
+        getenv("SCALANATIVE_INCLUDE_DIRS")
+          .map(_.split(" ").toSeq)
+          .getOrElse(Seq("/usr/local/include", "/opt/local/include"))
+
+      (includeDirs ++ llvmIncludeDir).map(s => s"-I$s")
     }
     includes :+ "-Qunused-arguments"
   }
 
   /** Find default options passed to the system's native linker. */
   def linkingOptions(): Seq[String] = {
     val libs = {
-      val libdir =
-        Try(Process("llvm-config --libdir").lineStream_!.toSeq)
+      val llvmLibDir =
+        Try(Process("llvm-config --libdir").lineStream_!)
```

Comment:
The `.toSeq` must stay. Otherwise exceptions thrown during reading will not be caught by the `Try`.